tests/flow/annot/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -149,6 +150,7 @@ function foo() {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 let myClassInstance: MyClass = null;
 // forward ref ok, null ~> class error
```

Comment:
This comment was actually on the same line originally, it's weird that it got moved to its own line